sprocs/instance_questions_calculate_stats.sql

```diff
@@ -15,6 +15,7 @@ WITH first_calculation AS (
         variants AS v
         JOIN submissions AS s ON (s.variant_id = v.id)
     WHERE v.instance_question_id = instance_question_id_param
+          AND s.gradable IS TRUE
```

Comment:
Nit for consistent formatting:
```suggestion
         AND s.gradable IS TRUE
```